pkg/rancher-desktop/pages/containers/logs/_id.vue

```diff
@@ -0,0 +1,648 @@
+
+          this.searchAddon = new SearchAddon();
+          this.terminal.loadAddon(this.searchAddon);
+
+          this.terminal.loadAddon(new WebLinksAddon((event, uri) => {
+            event.preventDefault();
+            window.open(uri, '_blank');
+          }));
+
+          this.terminal.open(this.$refs.terminalContainer);
+          this.fitAddon.fit();
+
+          if (this.pendingLogs) {
+            this.terminal.write(this.pendingLogs);
+            this.pendingLogs = '';
+          }
+          this.terminal.attachCustomKeyEventHandler((event) => {
+            if (event.key === '/') {
+              event.preventDefault();
+              if (this.$refs.searchInput) {
+                this.$refs.searchInput.focus();
+                this.$refs.searchInput.select();
+              }
+              return false;
+            }
+            return true;
+          });
+
+          window.addEventListener('resize', () => {
+            if (this.fitAddon) {
+              this.fitAddon.fit();
+            }
+          });
+        }
+      });
+    },
+    clearSearch() {
+      this.searchTerm = '';
+      if (this.searchAddon) {
+        this.searchAddon.clearDecorations();
+      }
+      this.$nextTick(() => {
+        if (this.$refs.searchInput) {
+          this.$refs.searchInput.focus();
+        }
+      });
+    },
+    performSearch() {
+      if (!this.searchAddon || !this.searchTerm) {
+        if (this.searchAddon) {
+          this.searchAddon.clearDecorations();
+        }
+        return;
+      }
+
+      try {
+        this.searchAddon.clearDecorations();
+
+        this.searchAddon.findNext(this.searchTerm);
+      } catch (error) {
+        console.error('Search error:', error);
+      }
+    },
+    searchNext() {
+      if (!this.searchAddon || !this.searchTerm) return;
+      try {
+        this.searchAddon.findNext(this.searchTerm);
+      } catch (error) {
+        console.error('Search next error:', error);
+      }
+    },
+    searchPrevious() {
+      if (!this.searchAddon || !this.searchTerm) return;
+      try {
+        this.searchAddon.findPrevious(this.searchTerm);
+      } catch (error) {
+        console.error('Search previous error:', error);
+      }
+    },
+    handleSearchKeydown(event) {
+      if (event.key === 'Enter') {
+        if (event.shiftKey) {
+          this.searchPrevious();
+        } else {
+          this.searchNext();
+        }
+        event.preventDefault();
+      } else if (event.key === 'Escape') {
+        this.clearSearch();
+        event.preventDefault();
+      }
+    },
+    handleGlobalKeydown(event) {
+      if (event.key === '/') {
+        event.preventDefault();
+        if (this.$refs.searchInput) {
+          this.$refs.searchInput.focus();
+          this.$refs.searchInput.select();
+        }
+      }
+    },
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@import '@xterm/xterm/css/xterm.css';
+
+:global(.body) {
```

Comment:
This applies when the component isn't loaded too.  As such, just modify the actual stylesheet instead of using `!important`?